src/EditorFeatures/CSharpTest2/Recommendations/AbstractKeywordRecommenderTests.cs

```diff
@@ -14,317 +14,317 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Recommendations
     public class AbstractKeywordRecommenderTests : KeywordRecommenderTests
     {
         [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        public async Task TestAtRoot_Interactive()
+        public void TestAtRoot_Interactive()
         {
-            await VerifyKeywordAsync(SourceCodeKind.Script,
+            VerifyKeyword(SourceCodeKind.Script,
```

Comment:
📝 I'm assuming that all changes to the tests are:

* Return type change
* Removing `Async` suffix

And as such am not reviewing these changes in much detail. If there are any behavior changes please call them all out or move them to a different PR.

Reply:
yup.  this was all done mechanically too using our own refactoring tools.